syntax = "proto3";
option go_package = "./;pw";

service WorkerForEdge {
  rpc Infer (InferRequest) returns (InferResponse) {}
}

message InferRequest {
  bytes content = 1;
}

message InferResponse {
  repeated ClassResult result = 1;
}

message ClassResult {
  int64 class = 1;
  repeated BoundingBox boxes = 2;
}

message BoundingBox {
  repeated double params = 1;
}